Formula/p/pistache.rb

```diff
@@ -0,0 +1,157 @@
+class Pistache < Formula
+  desc "Modern, fast, elegant HTTP + REST C++17 framework with pleasant API"
+  homepage "https://github.com/pistacheio/pistache"
+  url "https://github.com/pistacheio/pistache/archive/refs/tags/v0.4.26.tar.gz"
+  sha256 "29af6562547497acf6f49170661786fe8cf1ed3712ad80e69c53da4661c59544"
+  license "Apache-2.0"
+  head "https://github.com/pistacheio/pistache.git", branch: "master"
+
+  depends_on "cmake" => :build # Required so meson finds hinnant-date
+  depends_on "cpp-httplib" => :build
+  depends_on "googletest" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+  depends_on "brotli"
+  depends_on "howard-hinnant-date"
```

Comment:
this should be build dep?
```suggestion
  depends_on "howard-hinnant-date" => :build
```

Reply:
any thoughts about this?

Reply:
Yes, you're correct.

Now, for clarity, Pistache's _src/common/http_defs.cc_ has:
`#include <date/date.h>`
which comes from howard-hinnant-date. _http_defs.cc_ is part of the core Pistache library, not solely build or unit-test.
However, howard-hinnant-date is a header-only library - so once the build is done, the howard-hinnant-date library (i.e. header file) is no longer needed.

Changed.